src/planscape/plan/models.py

```diff
@@ -69,9 +69,6 @@ class Project(models.Model):
     # Ratio of elevation to distance
     max_slope: models.FloatField = models.FloatField(null=True)
 
-    # TODO: Add more project parameters like min_acres_treated and
-    # permitted_ownership = (1=federal, 2=state, 4=private)
-
 
 class Scenario(models.Model):
```

Comment:
just to double-check: does this correspond with RankingScenario in the db graph?

Reply:
yes!

Reply:
leaving these changes as is for now and we'll revise again with new DB adjustments